config/initializers/rack_attack.rb

```diff
@@ -18,7 +18,9 @@ def remote_ip
     # whitelisting). It must implement .increment and .write like
     # ActiveSupport::Cache::Store
 
-    Rack::Attack.cache.store = StoreProxy::RedisStoreProxy.new(Redis.current)
+    Rack::Attack.cache.store = StoreProxy::RedisStoreProxy.new(
+      Redis.new(url: Figaro.env.sidekiq_redis_url)
```

Comment:
I wonder if it's appropriate to separate rack attack and sidekiq into separate DBs in redis. It doesn't seem to make much of a performance difference (it's still single threaded). From what I can tell we only gain the guarantee of no key clashes and the ability to quickly wipe the individual DBs (`flushdb`). 

http://www.rediscookbook.org/multiple_databases.html